content/guest-post/opencv-web-app-with-streamlit/index.md

```diff
@@ -0,0 +1,226 @@
+---
+title: "How to Create a OpenCV Web App with Streamlit"
+date: "2021-11-01"
+coverImage: "coverimage.png"
+author: "Dingu Sagar"
+tags: ["OpenCV", "Streamlit", "Python"]
+description: "You can now easily publish your OpenCV apps to the web using Streamlit, a Python library. Follow this tutorial to learn the process step by step."
+---
+
+In this tutorial, you'll learn how to easily convert an OpenCV project into a web app that you can showcase. You'll use a library called Streamlit, which helps you easily build a web user interface in Python. Yes, you heard it right — no HTML, CSS, or JavaScript required. Just Python!
+
+## Prerequisites
+- You should be comfortable using basic [OpenCV](https://opencv.org/) functions.
+- You should be comfortable coding in Python.
+
+Let's get started.
+
+## Install Dependencies
```

Comment:
Along with dependencies, we can include a quick note regarding python version that is used in the writeup as well, in order to give context to the reader. It seems opencv has dropped support for python 2.7 from `opencv-python-4.3.0.36` (latest version is `4.3.0.38`), so legacy users will need to specify the opencv version to download. Python 3 users should be able to follow the guide without change.

Reply:
Sure @nathannguyenn,  have added the info related to this under the Install Dependencies section. please check .